superset/reports/schemas.py

```diff
@@ -208,10 +209,34 @@ class ReportSchedulePostSchema(Schema):
         dump_default=None,
     )
     force_screenshot = fields.Boolean(dump_default=False)
+    custom_width = fields.Integer(
```

Comment:
what about `custom_height`

Reply:
Custom height is not implemented, I added it to the model so we don't need to do another migration in the future.